server/lib/nl/detection/place.py

```diff
@@ -299,6 +312,23 @@ def get_place_from_dcids(place_dcids: List[str], debug_logs: Dict) -> any:
   return places, parent_places
 
 
+#
+# Helper function to get resolved entities that are not places.
+#
+def _get_non_place_entities(all_entities: List[str],
+                            resolved_places: List[Place]) -> List[Entity]:
+  entities = []
+  places = set([p.dcid for p in resolved_places])
+  non_place_entities = [e for e in all_entities if not e in places]
+  if non_place_entities:
+    names = property_values(non_place_entities, 'name')
+    for e in non_place_entities:
+      eNames = names.get(e, [])
```

Comment:
nit: skip camel case for variable names in py?

Reply:
done